### PR TITLE
Transfer from volute

### PR DESCRIPTION
**Pull request by pahjbo**
This is the intial transfer of the WD-1.1 document from volute - should be ready for working on GitHub from here

### CODE REVIEW

**Review by molinaro-m — APPROVED** [review]
The document import from svn to git compiles without errors.
I approve the PR to start working on its revision.

(I suggest to create tags to existing committed REC versions and add CI, that's included in ivoaTeX - but I leave it up to editors)

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+			-d "//feature[starts-with(form, 'ivo_hashlist_has')]" \
+			-d "//feature[starts-with(form, 'ivo_nocasematch')]" \
+			-d "//feature[starts-with(form, 'ivo_hasword')]" \
+		| xmlstarlet fo > $@.tmp
+#	gavo admin xsdValidate $@.tmp
+	sed -f samplegroom.sed $@.tmp > $@
+#	rm $@.tmp
+	
+install:
+	# local to Markus' setup
```
Comment (jd-au) [inline Makefile:50]: Should this section be here if it is local to gavo?

**Review by jd-au — APPROVED** [review]

File: Makefile (hunk 1 above)
Comment (msdemlei) [reply Makefile:50]: It should, yes.

Comment (pahjbo) [reply Makefile:50]: it's not part of the standard build target, but a way of generating the example - I think that it is a good idea in general to generate examples from reference implementations - even if it only remains as information as to how the example was generated, it is useful to future authors.

Comment (pahjbo) [reply Makefile:50]: ok talking about the wrong bit of the make - yes I guess that the install should go

**Comment by pahjbo** [comment]
this PR seems to have got a bit forgotten - it would be nice to get it merged so that further development of the WD can occur.

**Pull request merged**